.github/workflows/ams-ci.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run: google-chrome-stable --headless --disable-gpu --no-sandbox --remote-debugging-port=9222 http://localhost &
 
       - name: Prepare Test Environment
+        with:
+          node-version: '12.22.10'
```

Comment:
needed to set this to be the version that's in the package.json file to avoid the errors we were getting from the [github actions ci pipeline](https://github.com/WGBH-MLA/ams/runs/5453866535?check_suite_focus=true) (beginning at line 1105)

using node v`16.0.0` or higher and running `yarn` locally also resulted in errors, so I think node-sass legitimately has an issue with the [latest node version](https://nodejs.org/en/blog/release/v16.14.0/). 

resource: https://github.com/actions/setup-node#usage